src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -356,29 +357,40 @@ def attachDIDs(self, rse, superDID, dids, scope='cms'):
              then it's a container name; if attaching files, then it's a block name)
         :param dids: either a string or a list of data identifiers (can be block or files)
         :param scope: string with the scope name
+        :param chunkSize: number of dids to send to upstream Rucio server
```

Comment:
I would suggest to rephrase it to something like "maximum number of dids to be attached in any single Rucio server call"

Reply:
ok, done